src/main/scala/cromwell/engine/backend/runtimeattributes/CromwellRuntimeAttributes.scala

```diff
@@ -17,7 +17,8 @@ import scala.util.Try
 import scalaz.Scalaz._
 import scalaz._
 
-case class CromwellRuntimeAttributes(docker: Option[String],
+case class CromwellRuntimeAttributes(attributes: Map[String, WdlValue],
```

Comment:
Why have an attributes map AND the list of attributes? Can't you just have one or the other?